workspace/pkgs-graph/package.json

```diff
@@ -30,9 +30,10 @@
     "better-path-resolve": "1.0.0"
   },
   "dependencies": {
+    "@pnpm/logger": "^5.0.0",
```

Comment:
it should be added to peer dependencies.

Reply:
Cool, done